AppCenterAuth/AppCenterAuth/MSAuth.m

```diff
@@ -201,9 +224,19 @@ + (void)setConfigUrl:(NSString *)configUrl {
   [MSAuth sharedInstance].configUrl = configUrl;
 }
 
-- (void)completeSignInWithErrorCode:(NSInteger)errorCode andMessage:(NSString *)errorMessage {
-  if (!self.signInCompletionHandler) {
-    return;
+- (void)ifSignInExistsCompleteWithErrorCode:(NSInteger)errorCode
+                         andMessage:(NSString *)errorMessage
```

Comment:
We may be able to avoid passing the message again with the error refactoring: #1552

Reply:
I'm not sure I get how it affects passing messages. Besides, the PR is Data module refactoring and it doesn't introduce messages as constants.